MONITORING.md

```diff
@@ -85,3 +85,8 @@ the metrics on other one and see further instructions in next step.
    Full configuration example is available in [specs/spark-application.yaml](specs/spark-application.yaml).
 1) Now, go to the prometheus dashboard (e.g. `<kubernetes_endpoint_url>/ops/portal/prometheus/graph`) and search for metrics 
 starting with 'spark'. The Prometheus URI might be different depending on how you configured and installed the `prometheus-operator`. 
+
+1) Now it's an excellent time to get metrics visualized with Grafana. You can compose your dashboards or get started with these 
```

Comment:
from my view, it's better to be a little bit more official and concise here. Something like that:
```
KUDO Spark Operator comes with two dashboards:

  * [Spark Applications](/dashboards/grafana_spark_applications.json) which provides application-level metrics such as resources used per application, Spark internal metrics, and JVM metrics
<Spark Applications dashboards screenshot>
  * [Spark Operator](dashboards/grafana_spark_operator.json) which provides Operator metrics such as applications execution times, execution status etc.
<Spark Operator dashboards screenshot>
```

WDYT?

Reply:
addressed